package.json

```diff
@@ -18,12 +18,12 @@
   "dependencies": {
     "eventemitter2": "^6.4.9",
     "p-each-series": "^2.2.0",
-    "puppeteer": "^19.7.0"
+    "puppeteer": "^20.9.0"
```

Comment:
We already have the version 21.1.1 for Puppeteer 

https://www.npmjs.com/package/puppeteer